docs/swap.md

```diff
@@ -0,0 +1,183 @@
+Swap Service (regtest)
+=======================
+
+This docs shows you how to get the swap service up and running in your local dev environment. 
+
+The high level steps are:
+
+1. Start the loop server
+2. Configure the swap params in default.yaml
+3. Choose the swap provider in your code
+4. Test a swap out
+5. Monitor the status of a swap
+
+Quickstart
+----------
+```
+# 1. Start loop server
+make start-loop
+# 2. Monitor loop outs
+loopd1_id=$(docker ps -q -f name="loopd1-1")
+docker exec -it $loopd1_id loop -n regtest monitor
+# 3. Loop out
+TEST="swap-out" make integration
+# 4. Mine a block (or a few) to finish loop out
+make mine-block
+make mine-block
+```
+
+(1) Start the loopserver (regtest LL loop server) and loopd (Rest API)
+---------------------------------------
+```sh
+make start-loop
+```
+
+Successfully starting the loop server returns this:
+```json
+{
+  "swap_fee_sat":"50", 
+  "prepay_amt_sat":"1337", 
+  "htlc_sweep_fee_sat":"7262", 
+  "swap_payment_dest":"A31dVAIzvYn1p/0J7WG3DS7Qbq5FIzWhPgtHQngozskq",
+  "cltv_delta":0, 
+  "conf_target":9
+}
+```
+
+(2) Configure the loop params in `default.yaml`
+---------------------------------------
+
+```yaml
+swap:
+  minOnChainHotWalletBalance: 30000000
+  swapOutAmount:  25000000
+  swapProviders: ["LOOP"]
+  lnd1loopRestEndpoint: "https://localhost:8081"
+  lnd1loopRpcEndpoint: "localhost:11010"
+  lnd2loopRestEndpoint: "https://localhost:8082"
+  lnd2loopRpcEndpoint: "localhost:11011"
+```
+
+When testing in dev you want to override this config by creating a `/var/yaml/custom.yaml` file 
+
+create `/var/yaml/custom.yaml` and set permissions:
+```
+sudo touch /var/yaml/custom.yaml
+sudo chmod 755 /var/yaml/custom.yaml
+```
+
+Add the following config (you might need to 'retry as sudo' on mac+vscode):
+```
+swap:
+  minOnChainHotWalletBalance: 350000
+  swapOutAmount:  250000
```

Comment:
```suggestion
  minOnChainHotWalletBalance: 75000000
  swapOutAmount:  50000000
```

Reply:
I updated these values in the default.yaml. In this readme instruction I have the user configure a smaller value for the local custom config in `/var/yaml/custom.yaml` so that the integration tests run without running out of regtest bitcoin